### ISSUE TITLE
Karma and Todos for Ziv Izhar

### ISSUE THREAD

**Issue by zvili** [TechnionYP5777/Bugquery#11]
_From @ZivIzhar on October 30, 2016 19:18_

- [X]  Use a picture, photo or distinct avatar. Don't use an automatic icon
- [X]  Make sure you present yourself by a readable nick
- [X]  Describe yourself in the bio page;
- [X]  State that you are (not) in search for a job

_Copied from original issue: SpartanRefactoring/Spartanizer#644_

**Comment by zvili** [comment]
_From @ZivIzhar on October 30, 2016 19:24_

@yossigil Done. (accidently changed the name before mentioning)

**Comment by zvili** [comment]
_From @yonzarecki on October 30, 2016 19:39_

Good job :D

**Comment by zvili** [comment]
_From @yosefraisman on October 30, 2016 19:52_

Your page is the Chuck Norris of karma pages, Ziv. Well done!
If you'd like, please check out mine - it's not too shabby either! (#616)

**Comment by zvili** [comment]
_From @RonGatenio on October 30, 2016 22:52_

Nice page Ziv! Have some karma:  🎇 ❇️ 🌟 ✨  🌠  🎇 ❇️ 🌟 ✨  🌠 🎇 ❇️ 🌟 ✨  🌠

**Comment by zvili** [comment]
_From @roy-shchory on October 31, 2016 14:10_

came here to check if you were doing anything.
#ziv_do_something_with_your_self
if you want a guide for doing stuff go to #621

**Comment by zvili** [comment]
_From @Shahar-Y on October 31, 2016 14:26_

Your karma page is truely a masterpiece.
Visit mine at #665

**Comment by zvili** [comment]
_From @yossigil on November 26, 2016 11:51_

@zvili : please move to student's current repo

**Comment by yossigil** [comment]
Can you summarize here all issues you dealt with towards Sprint 1?

@AdiOmari

**Comment by ZivIzhar** [comment]
@yossigil , sure :)
most of the work was done with @tonylekhtman 
Issues summary until now:
- issue #9 
  - ramped up on sql and mysql
  - decided with tony the relevant fields from stackoverflow's given data
  - imported with tony data (on a smaller forum, since we don't have access to the server yet) into mysql
- issue #16 
  - ramped up on jdbc - using mysql through java
  - added with tony the relevant dependencies and code to the project

Since we were unfamiliar with sql, databases (,etc...) and how to work with them, a big part of the work was to learn how to do so.

**Comment by AdiOmari** [comment]
@ZivIzhar , Ziv please choose a time for the project concluding meeting [here](https://docs.google.com/document/d/1UvP8tNBX4N-4dRM6ZIFREnbjf9Yvwo3TecFZJmJoApg/edit?usp=sharing)
